web/src/compose_validate.ts

```diff
@@ -806,18 +806,49 @@ export function validate_message_length($container: JQuery): boolean {
     if (text.length > realm.max_message_length) {
         $textarea.addClass("flash");
         setTimeout(() => $textarea.removeClass("flash"), 1500);
+
+        const $banner_container = $("#compose_banners");
+        compose_banner.show_error_message(
+            $t({defaultMessage: "Message length shouldn't be greater than 10000 characters."}),
+            compose_banner.CLASSNAMES.missing_private_message_recipient,
```

Comment:
Add new classnames for the new errors instead of using `missing_private_message_recipient`